### PR TITLE
docs: proofread the introduction page

### PR DESCRIPTION
**Pull request by NickVolynkin**

### CODE REVIEW

File: gitbook-docs/README.md [hunk 1]
```diff
@@ -1,31 +1,54 @@
...
+
+1.  `clang` compiles the program into a circuit — an intermediate representation in bytecode.
+2.  `assigner` creates an assignment table using the compiled circuit and input data.
+
+### Zero-knowledge proof workflow with zkLLVM
+
+zkLLVM toolchain is a crucial part of the zero-knowledge proof verification workflow.
+Here's how it all works.
+
+1.  First, the companies who want to prove transactions develop their code in a compatible language, like C++.
```
Comment (hgedia) [inline gitbook-docs/README.md:29]: companies -> users

Comment (hgedia) [inline gitbook-docs/README.md:29]: `prove transactions` is incorrect and perhaps `prove computations` might be a better fit .

File: gitbook-docs/README.md [hunk 2]
```diff
@@ -1,31 +1,54 @@
...
+    They compile it with zkLLVM into a circuit — a provable bytecode representation of the application.
+    
+    {% hint style="info" %}
+    `=nil;` recommends using [Crypto3 library](https://docs.nil.foundation/crypto3/)
+    for developing most performant circuit components.
+    {% endhint %}
+
+1.  To prove a particular transaction, the proof requester
```
Comment (hgedia) [inline gitbook-docs/README.md:37]: 1. -> 2.
2.  In this step we run the computation and generate a proof for it. 
3. Proof is generated by proof producers & not by proof requesters
4. inputs of that transaction => inputs of that computation

File: gitbook-docs/README.md [hunk 3]
```diff
@@ -1,31 +1,54 @@
...
+    combines the circuit with the inputs of that transaction in the `assigner`
+    and gets two binary files:
+    
+    * Constraint, which represents the arithmetization of the circuit.
+    * Assignment table, which contains pre-processed public inputs.
+
+3.  Proof producers use these files to verify the proof.
```
Comment (hgedia) [inline gitbook-docs/README.md:44]: 1. Proof requesters are the users who will verify the proof. 
2. Proof verification is a binary operation true or false after which you would write your path-dependent business logic.

Comment (hgedia) [inline gitbook-docs/README.md:44]: I think we need to either define these roles of proof producer/requester or refer to them as users.

**Review by hgedia — CHANGES_REQUESTED** [review]

File: gitbook-docs/README.md (hunk 3 above)
Comment (NickVolynkin) [reply gitbook-docs/README.md:44]: We certainly need to define them. Will do it on this page.

File: gitbook-docs/README.md (hunk 2 above)
Comment (NickVolynkin) [reply gitbook-docs/README.md:37]: 1. In fact, this is an allowed and even optimal way to make numbered lists in markdown.

Comment (hgedia) [reply gitbook-docs/README.md:37]: I am referring to the sequence of numbering, it is 1. -> 1. -> 3 , not the formatting.

File: gitbook-docs/README.md (hunk 1 above)
Comment (nemothenoone) [reply gitbook-docs/README.md]: Fixed.

Comment (nemothenoone) [reply gitbook-docs/README.md]: Fixed.

**Comment by nkaskov** [comment]
Closed after long period of inactivity.

**Pull request closed without merging**